### PR TITLE
feat: [MP-08] Create reusable Flex container component

### PR DESCRIPTION
**Pull request by thisisofficial**
### Description

Created a reusable and customizable flex-container component.

Added it's storybook documentation.

image

### Testing Checklist

- [ ] Test cases have been added/updated
- [x] Tested locally in browser (at least one other than Chrome)
- [x] All tests pass

### What JIRA Ticket(s) does this PR address?

- [MP-123](https://cd95.atlassian.net/browse/MP-8)

### CODE REVIEW

File: src/components/flex/Flex.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    | 'flex-end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'start'
+    | 'end'
+    | 'left';
+  align?:
+    | 'stretch'
```
Comment (caarlosdamian) [inline src/components/flex/Flex.tsx:17]: Estos puedes hacer un type y exportarlo 

`export type alingTypes = 'flex-start' | 'flex-end';`

y llamarlo 

 align?: alingTypes,
 
 lo mismo con el de arriba

**Review by caarlosdamian — CHANGES_REQUESTED** [review]
one little comment

File: src/components/flex/Flex.tsx (hunk 1 above)
Comment (thisisofficial) [reply src/components/flex/Flex.tsx:17]: Fixing it now

**Review by caarlosdamian — APPROVED** [review]
lgtm

**Pull request merged**